tools/figma-inspector/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Figma to Slint",
```

Comment:
If this lives in `tools/figma-inspector`, then I think the plugin should also be called `Figma Slint Inspector` instead of `Figma to Slint` (the latter suggesting an export).

I'd go with `tools/figma-plugin` and call it `Figma to Slint (Beta)`.